src/operation/emulation_model.jl

```diff
@@ -504,10 +486,7 @@ Default solve method for an EmulationModel used inside of a Simulation. Solves p
   - `model::OperationModel`: operation model
   - `start_time::Dates.DateTime`: Initial Time of the simulation step in Simulation time.
   - `store::SimulationStore`: Simulation output store
-
-# Accepted Key Words
-
-  - `exports`: realtime export of output. Use wisely, it can have negative impacts in the simulation times
+  - `exports = nothing`: realtime export of output. Use wisely, it can have negative impacts in the simulation times
```

Comment:
Does this really slow things down significantly? I would expect data storage size to be a bigger issue.

Reply:
It depends on the size of the writes.